### ISSUE TITLE
Font support

### ISSUE THREAD

**Issue by sIRwa2** [still-scene/t3#42]
Support for .ttc .ttf .otf font formats.

maybe also support for Variable fonts, its the new hype / craze/ the kids go nuts for them!

**Comment by pixtur** [comment]
Yes. It would be great to add the MDSF font-create to the project. As far as I can tell, it has no external dependencies, so it should in theory be possible to import TTF.

For other effects it would awesome to generate outlines and meshes directly from TTF.

**Comment by sense-amr** [comment]
extremely important as a standard feature of any graphics program that deals with using fonts.. 
the process of having to use multiple node programs in order to convert fonts is basically something very few people are going to do and could easily be a thing that turns people off using T3

**Comment by sense-amr** [comment]
i wonder if this could be of assistance .. in getting standard font support into T3 
https://gitlab.freedesktop.org/freetype/freetype

**Comment by pixtur** [comment]
Yes, I already looked at freetype: This nuget-package looks promising: https://www.nuget.org/packages/SpaceWizards.SharpFont

But as I mentioned above, for 95% of all use cases in Tooll3 we want MSDF generate for ttf files.

[This nuget-package](https://www.nuget.org/packages/MsdfGen.Net/0.1.6-alpha) looks promising.

**Comment by domportera** [comment]
was looking into this and ran into these:
[msdf-atlas-gen](https://github.com/Chlumsky/msdf-atlas-gen)
[msdfgen](https://github.com/Chlumsky/msdfgen)

We could just drop the executables into the project and throw very simple cmd commands at it, then import the generated files

theoretically of course, i know very little about this but it seems like it would be easiest with these tools

**Comment by pixtur** [comment]
Actually this sounds like a good plan. I would be very happy if you could give it a try.

**Comment by domportera** [comment]
update: I tried this a while back and the JSON it outputs is missing a lot of details that our MSDF fonts have for some reason /:

**Comment by pixtur** [comment]
We should also check if we can revoke the sixfonts feature branch for tixl.

**Comment by pixtur** [comment]
Closing this issue, because with v4.1 we've added the build-in MSDF font converter.

However, we should create follow up features for:
- [ ] Dragging an ttf into project should open MSDF font converter
- [ ] Support loading ttf to generate 3d-geomoetry or point outlines
